tests/unit/design_document_tests.py

```diff
@@ -390,7 +390,8 @@ def test_fetch_dbcopy(self):
         # before comparison also. Compare the removed values with
         # the expected content in each case.
         self.assertEqual(db_copy, ddoc['views']['view002'].pop('dbcopy'))
-        self.assertEqual({'epi': {'dbcopy': {'view002': db_copy}}}, ddoc_remote.pop('options'))
+        self.assertEqual({'epi': {'dbcopy': {'view002': db_copy}}, 'partitioned': False}, ddoc_remote.pop('options'))
```

Comment:
Thanks for fixing this which was missed in #467.  Unfortunately this test doesn't get called in Travis since it only tests CouchDB.